spec/facility_spec.rb

```diff
@@ -23,4 +34,165 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#register_vehicle(cruz)' do
+    it 'registers a vehicle' do
+
+      @facility_1.add_service('Vehicle Registration')
+
+      expect(@facility_1.register_vehicle(@cruz)).to eq(@facility_1.registered_vehicles)
+    end
+  end
+
+  describe '#register_vehicle(bolt)' do
+    it 'registers (bolt)' do
+
+      @facility_1.add_service('Vehicle Registration')
+
+      expect(@facility_1.register_vehicle(@bolt)).to eq(@facility_1.registered_vehicles)
+    end
+  end
+
+  describe '#register_vehicle(camaro)' do
+
+    it 'registers (camaro)' do
+      @facility_1.add_service('Vehicle Registration')
+
+      expect(@facility_1.register_vehicle(@camaro)).to eq(@facility_1.registered_vehicles)
+    end
+  end
+
+  describe '#set_plate_type' do
+    it 'returns the plate type' do
+      @facility_1.registered_vehicles
+
+      expect(@facility_1.set_plate_type(@cruz)).to eq(:regular)
+      expect(@facility_1.set_plate_type(@bolt)).to eq(:ev)
+      expect(@facility_1.set_plate_type(@camaro)).to eq(:antique)
+    end
+  end
+
+  describe '#administer_written_test' do
```

Comment:
No concerns with these tests-- it looks like all outcomes are covered-- this is just some food for thought. 
In RSpec you can use `describe` blocks to organize and nest different scenarios. This can sometimes make large/complex tests easier to follow. For example: 
```
describe '#administer_written_test' do
  describe 'successful' do
    before(:each) do
      # setup for successful scenario
    end
    # example (it) blocks
  end

  describe 'unsuccessful' do
    before(:each) do
      # setup for successful scenario
    end
    describe 'registrant is too young' do
      # example block(s)
    end
    describe 'facility does not offer service' do
      # example block(s)
    end
  end
end
```
and so forth.